assets/js/util.js

```diff
@@ -6,6 +6,7 @@
 var SPACE_KEY_CODE = 32, ENTER_KEY_CODE = 13,
     HTTP_OK_CODE = 200, HTTP_BAD_REQUEST_CODE = 400,
     XHR_LOADING = 3, XHR_DONE = 4;
+var AMOUNT_SCROLLED_ON_WEBPAGE = 300;
```

Comment:
How about, `BACK_TO_TOP_ACTIVATION_HEIGHT`? Try combining it with the `var` on L11 as well.